src/DssSpell.sol

```diff
@@ -15,67 +15,93 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+
 pragma solidity 0.6.12;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
-interface OptimismL1GovernanceRelayLike {
-    function relay(address, bytes calldata, uint32) external;
-}
-interface ArbitrumL1GovernanceRelayLike {
-    function relay(address, bytes calldata, uint256, uint256, uint256, uint256) external;
+interface DssVestLike {
+    function create(address, uint256, uint256, uint256, uint256, address) external returns (uint256);
+    function file(bytes32, uint256) external;
```

Comment:
I think we don't need `file` in the current spell as the `cap` should have already been set before.